services/classroom/jupyterhub.md

```diff
@@ -5,7 +5,13 @@ icon: code-greater-than-or-equal
 ---
 # Introduction
 
-JupyterHub provides a convenient way to serve Jupyter Notebooks for multiple users with a pre-configured computing environment where users do not need to worry about installing any software packages. This set-up free environment is ideal for courses and workshops with computing components. If you are an instructor, CCV can provide access to a cloud-hosted JupyterHub for your class or workshop. The implementation is supported by Brown CIS, please follow the links below to request an instance for your class or search the documentation.
+Brown's JupyterHub service  provides a convenient, cloud-hosted way to serve Jupyter Notebooks for multiple users. Notebooks are launched within a pre-configured computing environment; users do not need to install any software packages. This set-up free environment is ideal for courses and workshops where instructors intend for students to begin coding with minimal obstacles. Jupyter’s flexibility allows instructors to pick the preferred language for a particular context, including Python, Julia, R and many more. Regardless of the language chosen, the Jupyter interface remains the same. To learn more about using Jupyter Notebooks for teaching and learning, [please visit this guide.](https://jupyter4edu.github.io/jupyter-edu-book/jupyter.html)
+
+On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and CIS support staff. 
+
+Once connected to JupyterHub servers, users enter an isolated workspace to write and run code. There are no time limit restrictions or specified lockout times, so feel free to use your personal JupyterHub server as needed and adhere to CCV's Computing Policies.
+
+If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown CIS; please follow the link below to request an instance for your class. CIS staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course  deployment. 
 
 <a href="https://docs.google.com/forms/d/e/1FAIpQLSct9rFCxLhPIezHI-RYRyEuSnvHrPZLMuUSFRTriIyd_3TAfA/viewform?usp=sf_link">Request a Hub</a> 
```

Comment:
if you want this to be a button I think you can throw `class="button is-primary"` (or whatever color you want) on the `a` tag

Reply:
Done! Thank you, @mcmcgrath13!!